Tasks/VersionAssemblies/versionAssemblies.ts

```diff
@@ -46,7 +48,7 @@ var separator = os.platform() === "win32" ? "\\" : "/";
 var buildRegexObj = new RegExp(buildRegex);
 if (buildRegexObj.test(buildNumber)) {
 	var versionNum = buildRegexObj.exec(buildNumber)[buildRegexIndex];
-	console.info(`Using prefix [${replacePrefix}] and version [${versionNum}] in folder [${sourcePath}]`);
+	console.info(`Using prefix [${replacePrefix}] and version [${versionNum}] and postfix [$(replacePostfix)] in folder [${sourcePath}]`);
```

Comment:
This should be using curly braces, not round braces (string interpolation). I've fixed it in the final merge.